polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.tsx

```diff
@@ -42,14 +42,14 @@ export function SecondaryAction({
   );
 
   return (
-    <span
+    <div
```

Comment:
Is this a breaking change? In general, I evaluate markup changes as breaking case by case, but was able to find `ActionMenu` overrides in `web` e.g.
`[class^='Polaris-ActionMenu-SecondaryAction'] a`
and
`[class^='Polaris-ActionMenu-SecondaryAction'] button`

Can the same affect be achieved by updating the styles to `display: block` or `display: inline-block`?

Reply:
I'm not sure how keeping it a span but applying `display: block` or `display: inline-block` mitigates any breaking changes any more than switching from a `span` to `div`? The change is the display and that will happen regardless of whether switching to a `div` or applying it through css. But maybe I'm missing something 🤔

Since it doesn't affect any component APIs I didn't think of it as a breaking change.

Reply:
> I'm not sure how keeping it a span but applying `display: block` or `display: inline-block` mitigates any breaking changes any more than switching from a `span` to `div`?

Because developers could be targeting the `span` has shown above with `a` and `button`.

Reply:
I'm being overly cautious since we don't expose a way for developers to style sub-elements (even though it's possible with vanilla CSS) and thus no mechanism for us to gather insights on the impact of these changes. If you are confident this change is not breaking I will resolve the conversation 👍

Reply:
Just chiming in here that our management of the internal layout of components  is private API. We explicitly do not want and lint against people overriding our class names, if they are reaching into private styles they're responsible for the risk taken as our internal styles can change at any time (class names included, though not relevant here) and are not considered a breaking change when they do.